app/api/status/rne/route.ts

```diff
@@ -0,0 +1,21 @@
+import { clientDocuments } from '#clients/api-rne/documents';
+import { verifySiren } from '#utils/helpers/siren-and-siret';
+
+export async function GET() {
+  try {
+    const dummySiren = verifySiren('552032534');
+
+    // uses actes for monitoring as they are rarely rate limited
+    await clientDocuments(dummySiren);
+    return new Response(JSON.stringify({ message: 'ok', status: 200 }), {
+      status: 200,
+      headers: { 'Content-Type': 'application/json' },
+    });
+  } catch (e: any) {
+    const status = e.status || 500;
+    return new Response(JSON.stringify({ message: 'ko', status }), {
+      status,
+      headers: { 'Content-Type': 'application/json' },
+    });
+  }
+}
```

Comment:
I dont think you need this. I factorized this into `ping-api-client` https://github.com/annuaire-entreprises-data-gouv-fr/site/blob/977ba72fceb272684f5101f1400f898b6b043153/app/api/ping/%5Bslug%5D/route.ts#L1